docs/src/content/tutorial/testing-contracts.md

```diff
@@ -111,90 +117,169 @@ describe("Transactions", function() {
 });
 ```
 
-### Full coverage
+### Reusing common test setups using fixtures
+
+The two tests that we've written begin with the necessary setup, which in this case means deploying the token contract. In more complex projects, this setup could involve multiple deployments and other transactions. Doing that in every test means a lot of code duplication. Plus, executing many transactions at the beginning of each test can make the test suite much slower.
 
-Now that we've covered the basics you'll need for testing your contracts, here's a full test suite for the token with a lot of additional information about Mocha and how to structure your tests. We recommend reading through.
+You can solve these problems by using **fixtures**. A fixture is a setup function that is run only the first time it's invoked. On subsequent invocations, instead of re-running it, Hardhat will reset the state of the network to what it was at the point after the fixture was initially executed.
```

Comment:
```suggestion
You can avoid code duplication and improve the performance of your test suite by **fixtures**. A fixture is a setup function that is run only the first time it's invoked. On subsequent invocations, instead of re-running it, Hardhat will reset the state of the network to what it was at the point after the fixture was initially executed.
```